### PR TITLE
plan: fix bug of stats count

### PR DESCRIPTION
**Pull request by hanfei1991**
In old time, I confuse the count of table scan and the count of selection after table scan. It doesn't affect the acurrency, but the show of count in explain.

### CODE REVIEW

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -547,6 +547,7 @@ func (ds *DataSource) convertToTableScan(prop *requiredProp) (task task, err err
 	}
 	statsTbl := ds.statisticTable
 	rowCount := float64(statsTbl.Count)
+	log.Warnf("rc1 %v", rowCount)
```
Comment (winoros) [inline plan/physical_plan_builder.go]: remove it.

File: plan/physical_plan_builder.go [hunk 2]
```diff
@@ -556,6 +557,7 @@ func (ds *DataSource) convertToTableScan(prop *requiredProp) (task task, err err
 			}
 			// TODO: We can use ds.getStatsByFilter(accessConditions).
 			rowCount, err = statsTbl.GetRowCountByIntColumnRanges(sc, pkCol.ID, ts.Ranges)
+			log.Warnf("rc2 %v", rowCount)
```
Comment (winoros) [inline plan/physical_plan_builder.go]: ditto

File: plan/physical_plan_builder.go [hunk 3]
```diff
@@ -576,6 +578,7 @@ func (ds *DataSource) convertToTableScan(prop *requiredProp) (task task, err err
 			selectivity = selectionFactor
 		}
 		rowCount = math.Min(prop.expectedCnt/selectivity, rowCount)
+		log.Warnf("rc2 %v", rowCount)
```
Comment (winoros) [inline plan/physical_plan_builder.go]: ditto

File: plan/cbo_test.go [hunk 4]
```diff
@@ -388,7 +388,7 @@ func (s *testAnalyzeSuite) TestOutdatedAnalyze(c *C) {
 	c.Assert(h.Update(dom.InfoSchema()), IsNil)
 	// FIXME: The count for table scan is wrong.
```
Comment (lamxTyler) [inline plan/cbo_test.go]: This can be removed.

**Comment by shenli** [comment]
Please fix the CI.

**Comment by shenli** [comment]
@hanfei1991 Please fix the CI.

**Comment by hanfei1991** [comment]
@zimulala yes, that's a furture optimization

**Comment by hanfei1991** [comment]
@winoros @lamxTyler PTAL

**Comment by zimulala** [comment]
@hanfei1991  Please fix CI.

**Pull request merged**